.travis.yml

```diff
@@ -29,7 +29,7 @@ before_install:
   - xcrun simctl list
   # Update the brew and build dependencies tools
   - brew outdated carthage || brew upgrade carthage
```

Comment:
Suggestion: You could try to skip this `brew outdated carthage || brew upgrade carthage`
https://blog.travis-ci.com/2019-02-12-xcode-10-2-beta-2-is-now-available